tap_dbt/streams.py

```diff
@@ -186,7 +188,12 @@ class RunsStream(AccountBasedIncrementalStream):
     name = "runs"
     path = "/accounts/{account_id}/runs"
     openapi_ref = "Run"
-    replication_key = "finished_at"
+    replication_key = "created_at"
```

Comment:
I don't think we can use this - see https://github.com/MeltanoLabs/tap-dbt/issues/213#issuecomment-1712972545